src/main/java/org/commcare/formplayer/session/FormSession.java

```diff
@@ -119,7 +125,7 @@ public FormSession(SerializableFormSession session,
     }
 
     public FormSession(UserSqlSandbox sandbox,
-                       FormDef formDef,
+                       FormDefinition formDefinition,
```

Comment:
the fact that `formDef` and `formDefinition` are different types here is very confusing.

Reply:
the other classes are called `SerializableFormSession` and `SerializableMenuSession`. I don't love the convention but maybe we should stick with it?

Reply:
Yeah I was concerned about this naming, and agree that sticking to convention is wise.

Reply:
See f209feb0bb30728edbbfc693a091904384750d35